tools/sense_studio/static/main.js

```diff
@@ -253,6 +253,21 @@ function toggleMakeProjectTemporal(path) {
 }
 
 
+function toggleShowPredictions(path, split, label) {
+    let logregInput = document.getElementById('logregInput');
+    response = syncRequest('/toggle-project-setting', {path: path, setting: 'show_logreg', split: split, label: label});
+
+    // Show/hide all LogReg prediction-labels
+    if (response.setting_status) {
+        logregInput.setAttribute('checked', 'checked');
+        $('.logreg-predictions').show();
```

Comment:
The logreg labels haven't been updated yet, so they are not displaying the outputs of the new logreg model that has just been trained

Reply:
The new method reloads the page to display the newly trained LogReg predictions.